adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerProxy.cs

```diff
@@ -454,6 +450,22 @@ private Bundle GetBrokerOptions(AuthenticationRequest request)
             return brokerOptions;
         }
 
+        private void ValidateBrokerRedirectURI(AuthenticationRequest request)
+        {
+            string computedRedirectUri = GetRedirectUriForBroker();
+
+            //During the silent broker flow, the redirect URI will be null.
```

Comment:
>//During the silent broker flow, the redirect URI will be null. [](start = 12, length = 63)

I assume you also have to ensure the computedUri should not be null? Which it likely isn't ... not sure if anything should be done here, but raising it as a question.

Reply:
im actually not sure what should happen if the computed redirect URI is null. If it is null, that could indicate that our app does not have to ability to get the redirect URI from the application. In that case, we would have no idea if the the user's redirect URI is correct. 

So the question I have is, should we stop the user's authentication request if ADAL cannot check the redirect URI?

My opinion is we should allow the request to continue because the user's redirect URI may very well be valid.